explorer/src/lib/components/list-item/ListItem.css

```diff
@@ -0,0 +1,64 @@
+.details-list__term {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.625rem;
+  padding: 0.9375rem 1.125rem;
+  background-color: var(--background-color-alt);
+  text-transform: uppercase;
+}
+
+.details-list__term:first-child {
+  border-top-left-radius: var(--control-border-radius-size);
+  border-top-right-radius: var(--control-border-radius-size);
+  padding: 1rem 1.125rem 0.9375rem;
+}
+
+.details-list__term:last-of-type {
+  border-bottom-left-radius: var(--control-border-radius-size);
+  border-bottom-right-radius: var(--control-border-radius-size);
+  padding: 0.9375rem 1.125rem 1rem;
+}
+
+.details-list__definition {
+  display: flex;
+  flex-direction: row;
+  gap: 0.625rem;
+  align-items: center;
+}
+
+@media (max-width: 640px) {
+  .details-list__term {
+    display: flex;
+    flex-direction: row;
+    gap: 0.625rem;
+    align-items: center;
+    padding: 0.25rem;
+    margin-bottom: 0.625rem;
+    border-radius: var(--control-border-radius-size);
+    position: relative;
+  }
```

Comment:
There is no need to overwrite repeating properties, such as display, align-items, etc. They don't change based on the screen with, so redeclaring them is redundant.